scripts/snapshot-tool.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+"""
+This script fetches the specified Konflux snapshot and shows the git commit information of the component builds.
+This helps you confirm that all components of the snapshot are built from a specific commit, which might not be the case if some builds failed.
+Additionally, it can update the `bundle-patch/bundle.env` file with the pullspecs from the specified snapshot.
+
+You can specify a snapshot with
+* `--snapshot <snapshot>`: fetch the selected snapshot
+* `--commit <commit>`: fetch the latest snapshot of a specified commit
+* by default, it fetches the latest snapshot triggered by a 'push' event.
+
+
+Usage:
+
+$ ./scripts/snapshot-tool.py --help
+usage: snapshot-tool.py [-h] [--snapshot SNAPSHOT | --commit COMMIT] [--update-bundle-pullspecs]
+
+options:
+  -h, --help            show this help message and exit
+  --snapshot SNAPSHOT   Fetch the specified snapshot by name
+  --commit COMMIT       Fetch the latest snapshot of the specified commit
+  --update-bundle-pullspecs
+                        Update the pullspecs in the bundle-patch/bundle.env file
+
+                        
+Example:
+
+$ ~/redhat/dist/konflux/konflux/scripts/snapshot-tool.py
+
+Snapshot tempo-t4pmt
+
+COMPONENT                COMMIT DATE           REVISION                                  COMMIT MESSAGE
+tempo-gateway            2025-01-15T13:31:43Z  5326dff5bf2854725b7b085c648082f4acfa5009  Fix CVE-2024-45338 and CVE-2024-45337 (#191)
```

Comment:
the component names changed. Now there is suffix `-main`

Reply:
updated